hacks/isolateVersion/6-updateCIWorkflows.sh

```diff
@@ -1,7 +1,7 @@
 notify "Updating CI workflows..."
 
-# 1. build-docs: remove everything after the line `NODE_OPTIONS: --max_old_space_size=4096`.
-sed -i '' '/NODE_OPTIONS: --max_old_space_size=4096/q' .github/workflows/build-docs.yaml
```

Comment:
This broke because I updated the heap size in `build-docs.yaml` recently. Here I've removed the heap size number entirely, in case we need to bump again in the future.